src/bccontext.rs

```diff
@@ -71,37 +71,6 @@ impl<'a> BitcodeContext {
         host_call(&self.request.id, stream, "Write", src)
     }
 
-    /// read_stream_chunked reads usize bytes ub chunks of chunksize from a fabric stream returning a slice of [u8]
-    /// # Arguments
-    /// * `stream_to_read`-  the fabric stream to read from
-    /// * `sz`-  usize size of bytes
-    /// * `chunksize`-  usize size of bytes to read in each chunk
-    /// # Returns
-    ///   byte slice of the read stream
-    /// [Example](https://github.com/eluv-io/elv-wasm/blob/019b88ac27635d5022c2211751f6af5957df2463/samples/objtar/src/lib.rs#L112)
-    ///
-    pub fn read_stream_chunked(&'a self, stream_to_read: String, chunk_size: usize) -> CallResult {
```

Comment:
Document on `read_stream` that setting a size of 0 results in the _entire_ stream being read into memory at once.